plugin/mmdb/README.md

```diff
@@ -0,0 +1,56 @@
+# mmdb
```

Comment:
wouldn't geo or geoip be more descriptive?

Reply:
I recognize I was on the fence choosing a name.

Technically speaking the plugin is providing support for `mmdb` format databases, some of which don't provide geoip data, see: Domain, ASN, connection type, ISP, anonymous IP, etc.

In the other hand `mmdb` databases seems to be better known and became popular providing geoip data, all of those databases (including the ones providing non geo data) are often referred as geoIP.

I really have no strong opinion on one or the other, at some point I even used geoIP2 as plugin name...then renamed to `mmdb` to be technically correct.

What do you think?

Reply:
Changed to geoip